src/test/java/net/minecraftforge/debug/DataGeneratorTest.java

```diff
@@ -798,7 +798,7 @@ protected void registerStatesAndModels()
             stairsBlock((StairBlock) Blocks.ACACIA_STAIRS, "acacia", mcLoc("block/acacia_planks"));
             slabBlock((SlabBlock) Blocks.ACACIA_SLAB, ForgeRegistries.BLOCKS.getKey(Blocks.ACACIA_PLANKS), mcLoc("block/acacia_planks"));
 
-            // TODO 1.19: fix fenceBlock, wallBlock, doorBlock, and co -SS
+            // TODO 1.19: fix fenceBlock, wall and co -SS
```

Comment:
it's still wallBlock so shouldn't be changed from the method name in the comment.
Additionally, would it be possible for you to look into these other cases and fix them at the same time? Would be a lot nicer all in one PR